conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -35,9 +35,9 @@ GET           /submissions/test-only/whats-in-session                  uk.gov.hm
 GET           /submissions/test-only/clear-session                     uk.gov.hmrc.gform.testonly.TestOnlyController.clearSession()
 GET           /submissions/test-only/config                            uk.gov.hmrc.gform.testonly.TestOnlyController.config()
 
-GET           /submissions/test-only/snapshot-share/:formTemplateId/:snapshotId/:maybeAcccessCode   uk.gov.hmrc.gform.testonly.TestOnlyController.restoreAll(formTemplateId: FormTemplateId, snapshotId: SnapshotId, maybeAcccessCode: Option[AccessCode])
-GET           /submissions/test-only/restore-all/:formTemplateId/:maybeAcccessCode/:snapshotId      uk.gov.hmrc.gform.testonly.TestOnlyController.restoreAllGet(formTemplateId: FormTemplateId, maybeAcccessCode: Option[AccessCode], snapshotId: SnapshotId)
-GET          /submissions/test-only/restore-continue/:formTemplateId/:maybeAcccessCode              uk.gov.hmrc.gform.testonly.TestOnlyController.restoreContinue(formTemplateId: FormTemplateId, snapshotId:String, maybeAcccessCode: Option[AccessCode])
+GET           /submissions/test-only/snapshot-share/:snapshotId                                     uk.gov.hmrc.gform.testonly.TestOnlyController.restoreAll(snapshotId: SnapshotId, maybAccessCode: Option[AccessCode])
```

Comment:
Typo `maybAccessCode`.

And please remove `-share`. Ticket show just an idea, not a hard requirement, the clearer the public link will be the better.

Reply:
done